modules/Hoot/managers/api.js

```diff
@@ -349,6 +349,27 @@ export default class API {
             .then( resp => resp.data );
     }
 
+    cancelUpload() {
+
+        let cancelURL = `${this.baseUrl}/ingest/ingest/upload`;
+
+        const source = axios.CancelToken.source();
+
+        axios.get( cancelURL, { cancelToken: source.token } )
```

Comment:
I don't think this will work.  You'll need to pass the cancel token to the original upload request, not issue a new request when cancel is called.  I think you'll want to pass the cancelToken to `Hoot.api.uploadDataset` in `importDataset.js`.  https://github.com/ngageoint/hootenanny-ui/blob/hoot2x/modules/Hoot/ui/modals/importDataset.js#L417

If you go back to the hoot2x branch version `source.cancel('Request canceled.');`  would be added here https://github.com/ngageoint/hootenanny-ui/blob/hoot2x/modules/Hoot/ui/modals/importDataset.js#L502